TheEthicalHackerCup/Assets/AttackMinigames/DDoSAttack/Scripts/SpawnFinishLine.cs

```diff
@@ -5,17 +5,25 @@
 public class SpawnFinishLine : MonoBehaviour
 {
     public GameObject finishLine;
-    public float spawnTime;
+    public int frameUntilSpawn;
+    private int startFrame;
+
+    void Start()
+    {
+        startFrame = Time.frameCount;
+    }
 
     // Update is called once per frame
     void Update()
     {
-        if(Time.fixedTime == spawnTime){
+        if(Time.fixedTime == startFrame + frameUntilSpawn)
```

Comment:
Replace fixedTime with frameCount and make framesUntilSpawn a private field that get's its value set in `Start()` instead of in the inspector so that it can be determined dynamically
`Time.frameCount == startFrame + frameUntilSpawn`